### PR TITLE
initialize players in turn order

### PR DESCRIPTION
**Pull request by bafolts**
Initializes players in turn order.

### CODE REVIEW

File: src/Game.ts [hunk 1]
```diff
@@ -156,46 +155,35 @@ export class Game implements ISerializable<SerializedGame, Game> {
       public id: string,
       private players: Array<Player>,
       private first: Player,
-      gameOptions?: GameOptions,
```
Comment (bafolts) [inline src/Game.ts:159]: Easier than dealing with an `undefined` for `gameOptions` I defined it in constructor. The generated javascript writes what we had for us.

File: src/Game.ts [hunk 2]
```diff
@@ -156,46 +155,35 @@ export class Game implements ISerializable<SerializedGame, Game> {
...
-    ) {
-      Database.getInstance();
```
Comment (bafolts) [inline src/Game.ts:161]: This shouldn't be needed as it returns a value we don't use.

File: src/Game.ts [hunk 3]
```diff
@@ -280,13 +268,15 @@ export class Game implements ISerializable<SerializedGame, Game> {
 
       corporationCards = this.dealer.shuffleCards(corporationCards);
 
+      // Failsafe for exceding corporation pool
```
Comment (chosta) [inline src/Game.ts:271]: you probably meant `exceeding`

File: src/Game.ts [hunk 4]
```diff
@@ -280,13 +268,15 @@ export class Game implements ISerializable<SerializedGame, Game> {
...
+      if (gameOptions.startingCorporations * players.length > corporationCards.length) {
```
Comment (chosta) [inline src/Game.ts:272]: I would probably throw an error here and made sure the frontend handles validation. But this doesn't hurt anyway.

**Review by chosta — APPROVED** [review]

**Pull request merged**